### PR TITLE
PRIME-763 - Multi-Site Registration

### PR DESCRIPTION
**Pull request by mtpultz**
* Combines PRIME-776, 754, 778, 779, 775, 783 (which sounds like a lot, but these tickets were redundant or small task within a greater whole)
* Also includes PRIME-757 for Remote Practitioners due this task being started before Site Registration, and needing to be merged into Site Registration so it can use all the current updates (only comprises 16 files of changes in this PR)

### CODE REVIEW

File: prime-angular-frontend/src/app/modules/site-registration/pages/organization-agreement/organization-agreement.component.html [hunk 1]
```diff
@@ -3,15 +3,14 @@
...
-  <!-- TODO refactor later -->
   <ng-container *ngIf="!hasAcceptedAgreement; else continue">
     <div class="organization-agreement"
          [innerHTML]="organizationAgreement | safeHtml"></div>
 
     <div class="footer mb-4">
       <mat-checkbox #accept>
         <div class="pl-3">
-          I have read and accept the Organization Agreement.
+          By checking here I agree to the terms of this Agreement as the duly authorized representative of Organization.
```
Comment (itchison) [inline prime-angular-frontend/src/app/modules/site-registration/pages/organization-agreement/organization-agreement.component.html:13]: "representative of the Organization" - ?

File: prime-angular-frontend/src/app/modules/site-registration/pages/remote-user/remote-user.component.html [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                </div>
+              </div>
+
+              <app-address [form]="remoteUserLocation.get('physicalAddress')"
+                           [formControlNames]="formControlNames"></app-address>
+
+            </section>
+
+            <ng-container appContextualContent>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Fuga possimus nostrum accusamus doloribus esse
```
Comment (itchison) [inline prime-angular-frontend/src/app/modules/site-registration/pages/remote-user/remote-user.component.html:80]: lorem ipsum surprises me here.

File: prime-angular-frontend/src/app/modules/site-registration/pages/technical-support/technical-support.component.html [hunk 3]
```diff
@@ -7,37 +7,15 @@
...
-          security mechanisms such as firewalls and anti-virus measures (both generally and those in use at the site.)
-        </li>
-      </ul>
-
-      This individual should be able to quickly provide a network diagram of the site, including peripherals, wireless
-      access points, connected devices, and all applications using the internet router(s). This individual may be the
-      same as the Signing Authority, the Administrator of PharmaNet Onboarding, or the Privacy Officer; however, they
-      must also possess the technical skill and understanding to answer complex network- and application-related
-      questions.
+      Technical Support Contact A site staff person, corporate IT or independent IT contractor, familiar with the
```
Comment (itchison) [inline prime-angular-frontend/src/app/modules/site-registration/pages/technical-support/technical-support.component.html:14]: "Technical Support Contact A site staff person" - this reads weird. Should the "A" be capitalized?

Comment (mtpultz) [reply prime-angular-frontend/src/app/modules/site-registration/pages/technical-support/technical-support.component.html:14]: Haha probably not, but this text was copy-pasted from provided text.

File: prime-angular-frontend/src/app/modules/site-registration/pages/remote-user/remote-user.component.html (hunk 2 above)
Comment (mtpultz) [reply prime-angular-frontend/src/app/modules/site-registration/pages/remote-user/remote-user.component.html:80]: yep, I'll check remove context component wrapper, and if they want it back they can provide text.

**Pull request merged**